backend/map/tests.py

```diff
@@ -1,3 +1,58 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from django.http import JsonResponse
+from map import get_user_all_locations_and_company
 from django.test import TestCase
 
-# Create your tests here.
+class TestMapFunctions(unittest.TestCase):
+    def setUp(self):
+        self.mock_user_transactions = [
+            {"latitude": 40.7128, "longitude": -74.0060, "merchant_name": "Company A"},
+            {"latitude": 34.0522, "longitude": -118.2437, "merchant_name": "Company B"}
+        ]
+
+        self.mock_esg_scores = {
+            "Company A": {"environment_score": 600},
+            "Company B": {"environment_score": 200}
+        }
+
+    @patch('/Users/chongwan/vscode/RATTM/backend/utils/firebase.py.db')
+    @patch('/Users/chongwan/vscode/RATTM/backend/login/use_cases.py')
```

Comment:
Is this supposed to be using Chong's path?

Reply:
Oh right! Let me look at the tests. Thanks for catching that!